### PR TITLE
Added 1.10 example for nginx

### PR DESCRIPTION
**Pull request by thakkaryash94**
Tested and verified NGINX package v1.10.3 for DC/OS 1.10 beta1 version.

### CODE REVIEW

File: nginx/1.10/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Click on `CONFIGURE` button
+- Enter "`hello-nginx-master`" in `CONTENTDIR` and "`https://github.com/mesosphere/hello-nginx/archive/master.zip`" in `CONTENTURL`
+![Catalog NGINX](img/nginx-config.png)
+- Click on `REVIEW AND DEPLOY`, verify values and click on `DOWNLOAD CONFIG.JSON` button from top right corner
+![Catalog NGINX](img/nginx-review.png)
+- Click on `DEPLOY` and wait
+- One popup should appear with `Success` message
+![Catalog NGINX](img/nginx-success.png)
+- Click on `Services` from left menu bar
+- there should be link button beside `nginx` service name, click on it or go to http://<MASTER-NODE-URL/ADDRESS>/service/nginx/
```
Comment (joerg84) [inline nginx/1.10/README.md]: s/there/There 
I will fix that during commit

**Comment by joerg84** [comment]
https://jira.mesosphere.com/browse/DCOS_COMMUNITY-136

File: nginx/1.10/README.md (hunk 1 above)
Comment (thakkaryash94) [reply nginx/1.10/README.md]: Oh, one typo. Thank you for noticing and reviewing.

**Comment by thakkaryash94** [comment]
I think now we can close issue #106 .

**Pull request merged**